Gradle.md

```diff
@@ -8,6 +8,8 @@ There are currently five experiments for Gradle. You could also perform these ex
 
 ## Requirements
 
+You must have [Develocity](https://gradle.com/develocity/) in order to use the Develocity Build Validation Scripts. At a minimum, your Develocity user must have the permissions `Publish build scans` and `View build scans and build data` granted explicitly or anonymously, as analyzing the results relies on the use of build scans.
```

Comment:
Given my comments above, I suggest rewriting this section and being explicit about what each permission is required for.